ios/departureWidget/Structs.swift

```diff
@@ -336,9 +336,8 @@ struct FavouriteDeparture: Codable {
     let quayName: String
     let quayPublicCode: String?
     let quayId: String
-    let stopId: String
 
-    init(id: String, lineId: String, lineName: String?, lineLineNumber: String, lineTransportationMode: TransportMode?, lineTransportationSubMode: TransportSubMode?, quayName: String, quayPublicCode: String, quayId: String, stopId: String) {
+    init(id: String, lineId: String, lineName: String?, lineLineNumber: String, lineTransportationMode: TransportMode?, lineTransportationSubMode: TransportSubMode?, quayName: String, quayPublicCode: String, quayId: String) {
```

Comment:
Just by looking at the code, it might be that this constructor `init` can be removed and the same in `DepartureLineInfo` there is no reference to it in other places unless is a mandatory init to have due to the decoder but we can try.

Reply:
I'm getting some errors when removing them, so it might require some more changes to make that work.